code/Events/Event.Grub.cs

```diff
@@ -0,0 +1,18 @@
+namespace Grubs;
+
+public static partial class GrubsEvent
+{
+	public static class Grub
+	{
+		public const string Damaged = "grub.damaged";
+
+		/// <summary>
+		/// Occurs when a grub is damaged.
+		/// <para><see cref="int"/> The network ident of the damaged grub</para>
```

Comment:
Just fix up the spacing here:
```suggestion
		/// <para><see cref="int"/>The network ident of the damaged grub</para>
```